### ISSUE TITLE
[Bug]: Can not download a real file with blob url from  playwright

### ISSUE THREAD

**Issue by zhongkeruanjiansunny** [microsoft/playwright#33972]
### Version

1.50

### Steps to reproduce

1. launch a  chrome by playwright
2. input  a url   "https://clever-elion-84f381.netlify.app/"   by hand
3. click download  (link: blob:https://clever-elion-84f381.netlify.app/3d09551b-18a1-42c0-941d-dd8996b62f13) by hand

### Expected behavior

chrome download a josn file.

### Actual behavior

chrome download a file, but zhe file not a real file,you can not find and open it.....
![Image](https://github.com/user-attachments/assets/0dbcce9f-f854-4b72-9d1f-965abdcd6642)

### Environment

```shell
windows11
chrome 131.0.6778.33
```

**Comment by yury-s** [comment]
Yeah, I can reproduce it, looks like the file is not actually written to the disk when playwright listens to downloads. The tests with downloads work just fine though, so it only affects manual interaction with the browser controlled by playwright.

```ts
import { test, expect } from '@playwright/test';
import fs from 'fs';

test('test', async ({ page }) => {
    const onDownloadPath = new Promise<string>(res => {
        page.on('download', dl => {
        void dl.path().then(res);
        });
    });
    await page.goto('https://clever-elion-84f381.netlify.app/');
    await page.click('a');
    const path = await onDownloadPath;
    expect(fs.readFileSync(path).toString()).toBe('{"foo":1,"bar":2}');
});
```

**Comment by pavelfeldman** [comment]
#### Why was this issue closed?

Thank you for your contribution to our project. This issue has been closed due to its limited upvotes and recent activity, and insufficient feedback for us to effectively act upon. Our priority is to focus on bugs that reflect higher user engagement and have actionable feedback, to ensure our bug database stays manageable.

Should you feel this closure was in error, please create a new issue and reference this one. We're open to revisiting it given increased support or additional clarity. Your understanding and cooperation are greatly appreciated.
